README.md

```diff
@@ -226,7 +226,7 @@ A few more notes:
   DOCKER_BUILD_OPTS=--build-arg UBUNTU_VERSION=18.04 --build-arg UBUNTU_NAME=bionic
   ```
 * If you run behind a proxy, you will have to configure the proxy,
-  e.g., for docker (`~/.docker/config.json`). See
+  e.g., for docker (`~/.docker/config.json`) and load the configuration inside the dev container by setting `DOCKER_DEV_RUN_OPTS += -v "~/.docker":"/root/.docker"` in `$FPC_PATH/config.override.mk`. See
   [Working from behind a proxy](#working-from-behind-a-proxy) below for more information.
```

Comment:
please review previous comments

Reply:
Let me re phrase my question. Is my configuration considered to qualify as "behind a proxy"?

Reply:
Well, I don't know - do you access the internet via a proxy?

Reply:
I would the the "normal" use case is no proxy ...

Reply:
right, so I don't have a proxy defined in my browser settings so I consider that my computer isn't "behind a proxy" so I shouldn't have applied the configurations?

Reply:
and since the issue is because of docker versions and not the fact that you are behind or not a proxy, should the comment be there or should it be another note?

Reply:
well, with this PR we won't have the issue anymore. It will be only a problem if someone loads the host docker configuration into the dev container. The only reason why someone would do it is because they need extra configuration for instance of proxy use.

Reply:
view comment in other thread.